Support AlmaLinux in Vulnerability Detector (#5904)
wazuh/wazuh-documentation: source/user-manual/capabilities/vulnerability-detection/offline-update.rst

        <update_interval>1h</update_interval>
    </provider>

AlmaLinux
---------

> Currently, the AlmaLinux vulnerabilities are fetched from one OVAL file for each supported AlmaLinux version.

Reviewer: ```suggestion
Currently, the module fetches AlmaLinux vulnerabilities from one OVAL file for each supported AlmaLinux version.
```

+-------------------+------------------------------------------------------------------------------------------------+
| OS                | Files                                                                                          |
+===================+================================================================================================+
| AlmaLinux 8       | `org.almalinux.alsa-8.xml <https://security.almalinux.org/oval/org.almalinux.alsa-8.xml>`_     |
+-------------------+------------------------------------------------------------------------------------------------+
| AlmaLinux 9       | `org.almalinux.alsa-9.xml <https://security.almalinux.org/oval/org.almalinux.alsa-9.xml>`_     |
+-------------------+------------------------------------------------------------------------------------------------+

> Feeds can be fetched from a custom repository using the `url` attribute for each supported OS, as shown in this example:

Reviewer: ```suggestion
To fetch feeds from a custom repository you can use the ``url`` attribute for each supported OS, as shown in this example:
```
